_tour/tour-of-scala.md

```diff
@@ -32,7 +32,7 @@ Scala is also a functional language in the sense that [every function is a value
 Furthermore, Scala's notion of pattern matching naturally extends to the [processing of XML data](https://github.com/scala/scala-xml/wiki/XML-Processing) with the help of [right-ignoring sequence patterns](regular-expression-patterns.html), by way of general extension via [extractor objects](extractor-objects.html). In this context, [for comprehensions](for-comprehensions.html) are useful for formulating queries. These features make Scala ideal for developing applications like web services.
 
 ## Scala is statically typed ##
-Scala is equipped with an expressive type system that enforces at compile-time that abstractions are used in a safe and coherent manner. In particular, the type system supports:
+Scala is equipped with an expressive type system that enforces that abstractions are used in a safe and coherent manner at compile-time. In particular, the type system supports:
```

Comment:
Maybe some commas would help to make the sentence 'flow' better:
```suggestion
Scala is equipped with an expressive type system that enforces, at compile-time, that abstractions are used in a safe and coherent manner. In particular, the type system supports:
```

Reply:
@Philippus I like that. Or, perhaps something like `Scala's expressive type system enforces, at compile-time, that abstractions are used in a safe and coherent manner.`?

Reply:
yeah, I agree that "is equipped with" doesn't really add meaning and might as well be dropped